vpc/backend.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  backend "s3" {
+    bucket         = "remote-state-bucket"
+    key            = "state-files/terraform.tfstate"
+    region         = "eu-north-1"
+    encrypt        = true
+  }
+}
```

Comment:
Can you add new line at the end of your file ?

Reply:
done